lib/useLocalStorage.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```

Comment:
Shouldn't hooks be in the `hooks` folder instead?

Reply:
I want to move the hooks folder inside the lib folder. What do you think?

Reply:
Either way's fine with me